lib/collection/src/shards/local_shard/clock_map.rs

```diff
@@ -181,11 +148,12 @@ pub enum Error {
     SerdeJson(#[from] serde_json::Error),
 }
 
-impl From<atomicwrites::Error<Error>> for Error {
-    fn from(err: atomicwrites::Error<Error>) -> Self {
+impl From<file_operations::Error> for Error {
+    fn from(err: file_operations::Error) -> Self {
         match err {
-            atomicwrites::Error::Internal(err) => err.into(),
-            atomicwrites::Error::User(err) => err,
+            file_operations::Error::Io(err) => err.into(),
+            file_operations::Error::SerdeJson(err) => err.into(),
+            _ => unreachable!(),
```

Comment:
Is this good enough, or should I convert these to `io::Error` still?

Reply:
Fine by me :+1: